components/motivators/security_net/SecurityNetAssistance.tsx

```diff
@@ -8,31 +8,56 @@ import { NavigationProp } from '@react-navigation/native';
 import { SecurityNetRoutes } from './SecurityNet';
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
 
-async function finishSecurityNetItem(navigation: NavigationProp<SecurityNetRoutes>, newComponent: SafetyNetDType) {
-  if (
-    newComponent.strategies[0] !== '...' &&
-    newComponent.strategies[1] !== '...' &&
-    newComponent.strategies[2] !== '...'
-  ) {
-    // TODO: send SafetyNetItem to DB with POST
-    navigation.navigate('SecurityNetHome');
-  }
-}
-
 export default function SecurityNetAssistance({
   navigation,
   route,
 }: NativeStackScreenProps<SecurityNetRoutes, 'SecurityNetAssistance'>) {
+  async function finishSecurityNetItem(navigation: NavigationProp<SecurityNetRoutes>, newComponent: SafetyNetDType) {
+    if (newComponent.strategies[0] !== '' || newComponent.strategies[1] !== '' || newComponent.strategies[2] !== '') {
+      if (modified) {
+        if (modifying) {
+          const response = await fetch(`http://localhost:4010/safetyNet/${newComponent.id}`, {
+            method: 'PUT',
+            headers: {
+              Authorization: 'Bearer react-native-app',
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(newComponent),
+          });
+        } else {
+          const response = await fetch('http://localhost:4010/safetyNet', {
+            method: 'POST',
+            headers: {
+              Authorization: 'Bearer react-native-app',
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(newComponent),
+          });
+        }
+      }
+      navigation.navigate('SecurityNetHome');
+    }
+  }
```

Comment:
Es wäre noch cool, wenn du die API-Calls in eine Klasse `SafetyNetClient extends BaseClient` kapselst. Kannst dich an WikiClient orientieren

Und die beiden `response` Variablen werden nicht verwendet

Reply:
Alles klar das mach ich dann kann ich damit auch direkt #89 fixen und Teil dieses PRs machen :)
Das mit der Texteingabe liegt wahrscheinlich an den Styles ich werf da nochmal nen Blick drauf